.travis.yml

```diff
@@ -2,14 +2,15 @@ dist: bionic
 env:
   global:
     - TZ=America/Los_Angeles
+language: python
 python:
   - '3.9'
 services:
   - postgresql
 sudo: required
 install:
   - pip install 'urllib3[secure]'
-  - sudo pip install -r requirements.txt
+  - pip install -r requirements.txt
```

Comment:
I think it was failing the build.  If I recall, the build was failing due to using the wrong python version even though it was configured in travis.yml to be version 3.9.   This might have been what fixed it.  Also it's best to not go beyond 3.9 right now.   We need to update our libraries first and we are very outdated.  I have an upgrade that depends on this PR, so if we can avoid making more significant changes to this PR, it will help.

Reply:
I don't see how removing sudo would fix a version problem.  On the other hand I don't know why the sudo is there.